fineract-provider/src/main/java/org/apache/fineract/cob/service/InlineLoanCOBExecutorServiceImpl.java

```diff
@@ -241,7 +241,7 @@ private boolean isLockOverrulable(LoanAccountLock loanAccountLock) {
         if (isBypassUser()) {
             return true;
         } else {
-            return StringUtils.isNotBlank(loanAccountLock.getError());
+            return StringUtils.isBlank(loanAccountLock.getError());
```

Comment:
I dont think it is correct to do this. Please undo this change. We can change the lock if we are going with bypass user or there was an error stored on the lock.

Reply:
Done! Change undone